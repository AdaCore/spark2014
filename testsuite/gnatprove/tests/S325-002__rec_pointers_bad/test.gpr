project Test is

   for Languages use ("Ada");

   package Compiler is
      for Default_Switches ("ada") use ("-gnata");
   end Compiler;

end Test;

