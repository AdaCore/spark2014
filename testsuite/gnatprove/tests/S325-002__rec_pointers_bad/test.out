incomplete_type_auto.ads:12:09: "C" is not allowed in SPARK (due to incomplete type with deferred full view)
incomplete_type_auto.ads:12:09: "C" is not allowed in SPARK (due to incomplete type with deferred full view)
incomplete_type_auto.ads:12:09: access to incomplete type "C_Acc" is required to be in SPARK
incomplete_type_auto.ads:12:09: access to incomplete type "C_Acc" is required to be in SPARK
incomplete_type_auto.ads:13:09: incomplete type with deferred full view is not yet supported
incomplete_type_auto.ads:14:09: "D" is not allowed in SPARK (due to incomplete type with deferred full view)
incomplete_type_auto.ads:14:09: access to incomplete type "C_Acc" is required to be in SPARK
gnatprove: error during flow analysis and proof
